src/player/player.c

```diff
@@ -263,6 +263,54 @@ void playerSetGrabbing(struct Player* player, struct CollisionObject* grabbing)
     } else if (grabbing != player->grabConstraint.object) {
         pointConstraintInit(&player->grabConstraint, grabbing, 8.0f, 5.0f, 1.0f);
     }
+    
+    if (player->grabConstraint.object) {
```

Comment:
Having a separate function that is only called at one place to replace a block of code is not necessarily "more organized". It looks prettier and can help with readability at times, I didn't feel like this was needed here but I made the change nonetheless. The lookup table is a good call.

Reply:
This is true in general - it's possible to go too far. But by the same token it's not necessarily better to avoid factoring out independent logic just because it is called once.

Like most things it varies by case. Here, I think it's warranted given the code size and specific/self-contained purpose. It doesn't have to be perfect, but performance being equal, readability and maintainability are as important as function.